Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md

```diff
@@ -36,3 +36,4 @@ Other
 | Issue      |
 |------------|
 | **Date Added:** 2018-12-04 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Parallel creation of collections using multiple client connections with the same database user may spuriously fail with "Could not update user due to conflict" warnings when setting user permissions on the new collections. A follow-up effect of this may be that access to the just-created collection is denied. <br> **Affected Versions:** 3.4.0 <br> **Fixed in Versions:** 3.4.1 <br> **Reference:** [arangodb/arangodb#5342](https://github.com/arangodb/arangodb/issues/5342)  |
+| **Date Added:** 2019-02-18 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** There is a clock overflow bug within Facebook's RocksDB storage engine for Windows. The problem manifests under heavy write loads, including long imports. The Windows server will suddenly block all writes for minutes or hours, then begin working again just fine. An immediate workaround is to change the server configuration: <pre><code>[rocksdb]&#13;throttle = false</code></pre> **Affected Versions:** all 3.x versions (Windows only) <br> **Fixed in Versions:** 3.4.4 <br> **Reference:** [facebook/rocksdb#4983](https://github.com/facebook/rocksdb/issues/4983)  |
```

Comment:
one question is this affecting 3.3 as well? if yes we will need to add the known issue also in the 3.3 page of this 3.4 branch (as unresolved, if the fix is not in 3.3, e.g. "-")

Reply:
This impacts every version of arangodb that includes rocksdb.  Frank has merged fixes to devel and 3.4 branches.

Reply:
okay, thanks - then we have to mention to issue in this page as well - https://docs.arangodb.com/3.4/Manual/ReleaseNotes/KnownIssues33.html

Reply:
Your team doing the PR, or would prefer I do it?

Reply:
should be an easy push in this PR. No preferences: anyone who is available and can/want